thatdevhuddy.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "@thatdevhuddy"
+project: "PianoBot"
+---
+
+# PianoBot
+
+## Summary
+
+We are going to build a self-playing piano. Our team has 4 members. One of our group member’s has a brother who is very skilled at playing the piano. Although we do not possess the same talent as he does, we are determined to mimic his abilities through innovation and technology. Our design consists of solenoids placed on top of every key (we’re only doing 2 octaves to start) that is controlled by a Teensy that reads a MIDI roll and pushes the solenoids accordingly. We are not following any guides.
+
+## Plan
+
+Prototype Circuit with LEDs
+Write code to read MIDI roll and turn on and off LEDs accordingly
```

Comment:
What might this involve? Could you specify programming languages/frameworks/etc?

Reply:
@ThatDevHuddy: I see that you have marked this comment as resolved; were you able to update your file with these details?